packages/components/button/src/use-button.ts

```diff
@@ -223,6 +222,7 @@ export function useButton(props: UseButtonProps) {
     spinnerSize,
     disableRipple,
     getButtonProps,
+    getRippelProps: () => ({ripples, onClear: onClearRipple}),
```

Comment:
```suggestion
    getRippleProps: () => ({ripples, onClear: onClearRipple}),
```

Reply:
@li-jia-nan please create the function `getRippleProps` using `useCallback` so we avoid creating it on every render